scipy/stats/stats.py

```diff
@@ -3692,6 +3692,103 @@ def f_oneway(*args, axis=0):
     return F_onewayResult(f, prob)
 
 
+AlexanderGovernResult = namedtuple("AlexanderGovernResult", ("statistic",
```

Comment:
Make a bare bones class with just fields `statistic` and `pvalue` for now. E.g. CramerVonMisesResult in gh-11119. `namedtuple` is not flexible enough.

Reply:
@WarrenWeckesser indicated in gh-11119 that he wants a `__repr__` method, too:
```
class CramerVonMisesResult:
    def __init__(self, statistic, pvalue):
        self.statistic = statistic
        self.pvalue = pvalue

    def __repr__(self):
        return (f"{self.__class__.__name__}(statistic={self.statistic}, "
                f"pvalue={self.pvalue})")
```
Please adapt to your purposes. I think this means copy paste it into your code and change the name of the class to AlexanderGovernResult.

Reply:
Added